scripts/pylib/twister/twisterlib/package.py

```diff
@@ -27,10 +27,11 @@ def package(self):
         with open(os.path.join(self.options.outdir, "twister.json"), "r") as json_test_plan:
             jtp = json.load(json_test_plan)
             for t in jtp['testsuites']:
-                if t['status'] != TwisterStatus.FILTER:
-                    p = t['platform']
-                    normalized  = p.replace("/", "_")
-                    dirs.append(os.path.join(self.options.outdir, normalized, t['name']))
+                for tc in t['testcases']:
```

Comment:
isn't the problem actually because of missed TestSuite status for this use case after the 'NOTRUN' introduced ?

Reply:
I'd personally start with changing 'naked' key access (`t['testcases']`) into a `.get()` safe access (`t.get('testcases')`), thus eliminating the possibility of a Twister crash.

To fix this, one would need to add `TwisterStatus.NOTRUN` to this `if`-`elif` chain: https://github.com/LukaszMrugala/zephyr/blob/abfeb6a6a937be55672b0cf7733282b97ec28736/scripts/pylib/twister/twisterlib/reports.py#L337-L356

I'll write this in the relevant issue.

Reply:
I add